packages/lib-classifier/src/plugins/drawingTools/experimental/components/LineControls/LineControls.js

```diff
@@ -35,7 +35,7 @@ const LineControls = forwardRef(function LineControls({
   const { canvas, rotate } = useContext(SVGContext)
 
   // Get the transformation that's been applied to a parent element
-  const { matrix } = canvas.parentElement.parentElement.transform.baseVal.consolidate()
+  const matrix = canvas.parentElement.parentElement.getCTM()
```

Comment:
`canvas.parentElement.parentElement` is very brittle here. 😞 Any change to the structure of the subject viewer DOM would break the freehand line tools.